### ISSUE TITLE
Far Manager does not support Google Drive shortcuts

### ISSUE THREAD

**Issue by Korb** [FarGroup/FarManager#418]
In Google Drive, the user can create links to files, Google documents and folders. Interface for creating a shortcut using the context menu: ("Add a shortcut to Drive. Shortcuts link to the original, allowing an item to appear in more than one location."). Windows Explorer and Total Commander 10 open folder links without any problems. Far Manager 3.0.5822.0 x64 displays links such as .lnk files, and opens them in the browser instead of displaying the content.

Microsoft Windows 10.0.19042.1055.

**Comment by alabuzhev** [comment]
Is it specific to Google Drive at all?
.lnk files are just Windows shortcuts.

**Comment by Korb** [comment]
Yes, .lnk files are Windows shortcuts. This is how the Google Drive app displays links to googledrive' files and folders. And, as I wrote above, they work properly in Explorer and TC, and do not work in FAR.

![Create a shortcut to a folder in Google Drive](https://user-images.githubusercontent.com/3509797/122468612-9c6f1f80-cfc4-11eb-823d-2e610eb8eac1.png) ![A shortcut to a folder in Google Drive](https://user-images.githubusercontent.com/3509797/122468614-9da04c80-cfc4-11eb-91bc-d06f722167a7.png) ![Shortcut to a folder in Far 3 0 5822 0 x64](https://user-images.githubusercontent.com/3509797/122468615-9da04c80-cfc4-11eb-9933-6f68b9dfea01.png) ![Shortcut to a folder in Total Commander 10](https://user-images.githubusercontent.com/3509797/122468617-9da04c80-cfc4-11eb-8bdd-7bef54b47e21.png) ![Shortcut to a folder in Windows 10 Explorer](https://user-images.githubusercontent.com/3509797/122468619-9e38e300-cfc4-11eb-9852-623ccd1bb9f2.png)

**Comment by bitraid** [comment]
This is expected if you don’t have a plugin for that file format (Far itself does not parse the contents of files, and there is not an official plugin for `.lnk`). Use the [LnkView](https://plugring.farmanager.com/plugin.php?pid=952&l=en) plugin.

**Comment by Korb** [comment]
> Use the [LnkView](https://plugring.farmanager.com/plugin.php?pid=952&l=en) plugin.

lnkview-4.0.1-far3.0.0.4040-x86-64.rar, "Explore LNK-file" and "Follow shortcut" (the same result):

```
╔════════ LnkViewer ════════╗
║ Error reading LNK-file... ║
╟───────────────────────────╢
║          { OK }           ║
╚═══════════════════════════╝
```
```
 ╔═ ErrCode=00000000 ═╗
 ╟────────────────────╢
 ║       { OK }       ║
 ╚════════════════════╝
```
